charts/longhorn/100.1.1+up1.2.3/Chart.yaml

```diff
@@ -6,7 +6,7 @@ annotations:
   catalog.cattle.io/namespace: longhorn-system
```

Comment:
@shuo-wu same here, I can add `catalog.cattle.io/kube-version: '>=1.18.0-0 < 1.22.0-0'` to it.

Reply:
yes I believe this supports 1.22